Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,250 @@
+
+            Console.WriteLine($"{fullName} is {age} years old, his weight is {weightInKg} kg and his height is {heightInCm} cm.");
+            Console.WriteLine($"His BMI is {CalculateBmi(weightInKg, heightInCm / 100)}.");
+        }
+
+        /// <summary>
+        /// Prompts for an integer value. Converts the string input to an integer if possible.
+        /// </summary>
+        /// <param name="message">The message to print before the prompt.</param>
+        /// <param name="result">The parsing result. Is 0 if the parsing failed.</param>
+        /// <param name="input">The user input.</param>
+        /// <returns>Returns true if the parsing succeeded, false otherwise.</returns>
+        private static bool PromptTryParse(string message, out int result, out string input)
+        {
+            input = PromptInput(message);
+            return int.TryParse(input, out result);
+        }
+
+        /// <summary>
+        /// Prompts for a float value. Converts the string input to a float if possible.
+        /// </summary>
+        /// <param name="message">The message to print before the prompt.</param>
+        /// <param name="result">The parsing result. Is 0 if the parsing failed.</param>
+        /// <param name="input">The user input.</param>
+        /// <returns>Returns true if the parsing succeeded, false otherwise.</returns>
+        private static bool PromptTryParse(string message, out float result, out string input)
+        {
+            input = PromptInput(message);
+            return float.TryParse(input, NumberStyles.Float, CultureInfo.InvariantCulture, out result);
+        }
+
+        /// <summary>
+        /// Prints the message and reads in the user input.
+        /// </summary>
+        /// <param name="message">The message to print.</param>
+        /// <returns>The unparsed and unvalidated input.</returns>
+        private static string PromptInput(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Returns a value that indicates the type of input fault./>
+        /// </summary>
+        /// <param name="input">The input the user wrote.</param>
+        /// <returns>An integer indicating the error.</returns>
+        private static int InvalidInput(string input)
+        {
+            if (string.IsNullOrEmpty(input))
+            {
+                return EmptyNumberInput;
+            }
+            else
+            {
+                return PrintInvalidNumber(input);
+            }
+        }
+
+        /// <summary>
+        /// Checks the validity of the value and returns the (fixed) value accordingly.
+        /// </summary>
+        /// <param name="value">The value to check.</param>
+        /// <returns>Returns the value if it is in range or a value indicating the opposite.</returns>
+        private static int FixRange(int value)
+        {
+            if (value > 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885761)